lib/rules.rb

```diff
@@ -0,0 +1,11 @@
+class Rules
+  MOVES = [:rock, :paper, :scissors]
+  BEATS = { rock: :scissors, paper: :rock, scissors: :paper }
+
+  attr_reader :moves, :beats
```

Comment:
Btw, if you create a method `beats?` as suggested before, you'll see you don't need anymore those instance variables and `attr_reader`s :)